pkg/translate/service_test.go

```diff
@@ -4,38 +4,49 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/brianvoe/gofakeit/v6"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.expect.digital/translate/pkg/model"
 	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
 )
 
 // ----------------------GetService Parse Params-------------------------------
 
+func randGetServiceReq() *translatev1.GetServiceRequest {
+	return &translatev1.GetServiceRequest{
+		Id: gofakeit.UUID(),
+	}
+}
+
 func Test_ParseGetServiceParams(t *testing.T) {
 	t.Parallel()
 
+	happyReq := randGetServiceReq()
+
+	malformedIDReq := randGetServiceReq()
+	malformedIDReq.Id += "_FAIL" //nolint:goconst
```

Comment:
Switched to that kind of declaration for test inputs

- Table tests are shorter and more concise
- Random data is generated, which could potentially point to some uncovered edge case